components/tokens/index.js

```diff
@@ -78,7 +78,16 @@ const getVotingPowerAtLevel = async (req, response) => {
       userAddress
     );
 
-    response.json({ votingWeight });
+    const votingXTZWeight = await getUserTotalVotingPowerAtReferenceBlock(
+      network,
+      address,
+      daoContract,
+      tokenID,
+      level,
+      userAddress
```

Comment:
shouldn't this have isXTZ = true ?

Reply:
As specified in the task: "If isXTZ is true for any given poll, everything else remains the same, but for the vote action, the app queries a different endpoint on tzkt.io that returns the user's XTZ balance instead of the balance of the governance token."

Meaning, the user only requires to have some of the DAO's token in order to create an off-chain poll (It's XTZ balance is not important for this).

Therefore, there's no need for us to know the user's XTZ balance. If we took into consideration the user's XTZ balance in order to create an off-chain poll, any user (not a member of the DAO) would be able to create a poll for any DAO

Reply:
Okay, If that's the case it should be fine. Do check though if this function is being used in other places don't cause any side effects on the frontend.